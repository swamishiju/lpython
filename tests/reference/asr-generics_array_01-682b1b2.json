{
    "basename": "asr-generics_array_01-682b1b2",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/generics_array_01.py",
    "infile_hash": "6e943dd0e26ab4d1ffb6ed6909a365b4135b6f5295957b2478cfb479",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-generics_array_01-682b1b2.stderr",
    "stderr_hash": "15749848f430f565cf4c8500f45a67a0d405bddc6b0459ccad6d25f2",
    "returncode": 2
}