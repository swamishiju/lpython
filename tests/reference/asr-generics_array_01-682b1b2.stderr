semantic error: Type mismatch in procedure call; the types must be compatible
  --> tests/../integration_tests/generics_array_01.py:15:13
   |
15 |     print(f(array, x))
   |             ^^^^^ type mismatch (passed argument type is i32[1] but required type is T[:])
   |
 6 | def f(lst: T[:], i: T) -> T:
   |            ^^^^ type mismatch (passed argument type is i32[1] but required type is T[:])
